templates/style.scss

```diff
@@ -811,3 +811,8 @@ i.dependencies.normal {
   visibility: hidden;
   display: none;
 }
+
+/* Don't put a newline after code fragments in headers */
+h3 > code {
+  display: inline;
```

Comment:
Just wondering: why `inline` and not `inline-block`?

Reply:
I had to look up the difference - `inline-block` prevents newlines within the code snippet, while `inline` would allow line wrapping? In that case we should probably do inline-block.

https://stackoverflow.com/a/45061654/7669110